hydromt_wflow/naming.py

```diff
@@ -770,15 +770,15 @@
         "wflow_v0": "vertical.rhos",
```

Comment:
I'm not sure if it's a merge conflict or something we forgot but this is now just one variable in wflow.jl. So you can delete this one and rename the sediment_density_river one below to sediment_density